tribuo-classification/client/build.gradle

```diff
@@ -10,10 +10,8 @@ application {
 
 dependencies {
     implementation project(":common")
-    implementation ("org.tribuo:tribuo-all:4.1.0@pom") {
-        transitive = true // for build.gradle (i.e., Groovy)
-        // isTransitive = true // for build.gradle.kts (i.e., Kotlin)
-    }
-    implementation "info.picocli:picocli:4.6.1"
+    implementation ("org.tribuo:tribuo-classification-core:$tribuoVersion")
+
+    implementation "info.picocli:picocli:$picocliVersion"
     implementation "com.r3.conclave:conclave-web-client:$conclaveVersion"
 }
```

Comment:
Can you add a newline at the end of this file, please?

Reply:
done